src/actions/bityActions.js

```diff
@@ -144,7 +146,7 @@ export const getEstimate = (fiat: string, history: Object) => async (dispatch: D
     const outputFix = {
       currency: 'EUR'
     }
-    const payload1 = {input: inputFix, output: outputFix}
+    const payload1 = {input: inputFix, output: outputFix, PARTNER_FEE}
     const fiatDividerEst = await apiEstimate(payload1)
```

Comment:
Maybe modify the `apiEstimate` function itself with this data.

Reply:
Also, you probably want `{ partner_fee: PARTNER_FEE }`, since they expect the field in lowercase.